app/controllers/concerns/idv_session.rb

```diff
@@ -26,16 +28,23 @@ def idv_session
   protected
 
   def idv_attempts=(num)
-    user_session[:idv_attempts] = num
+    current_user.update!(idv_attempts: num)
   end
 
   def idv_attempts
-    user_session[:idv_attempts] ||= 0
+    current_user.idv_attempts ||= 0
+  end
```

Comment:
That's a very good point. Something about having getters and setters in the controller seems off to me, though. I'm not sure why. Maybe it's because this is the first time I've seen it. I'll have to think about this some more later.